server/server.go

```diff
@@ -128,6 +131,10 @@ func NewServer(ctx *Context, stopper *util.Stopper) (*Server, error) {
 			return nil, err
 		}
 	}
+
+	//
+	s.sqlDB = sqlserver.NewDBServer(s.db)
```

Comment:
There is a bit of stuttering going on with `server` in both the package name and method name. Perhaps the code in `sql/sqlserver` should live in `sql`. Not sure what `DB` means in this context (this comment applies to `kv.DBServer` as well). `sql.Server` would be a good final name for the type.